controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -6743,23 +6744,25 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
         const billingRepo = new BillingRepository(opts.db);
         const plans = await billingRepo.listPlans();
 
-        if (!plans?.length) {
-          return {
-            response: {
-              code: EnumStatusCode.ERR,
-              details: 'No billing plans configured. Please contact support.',
-            },
-          };
-        }
+        if (opts.stripeSecretKey) {
```

Comment:
This condition is wrong. The method should not fail when no billing is configured.

Reply:
no , it checks if the plans exist only when the billing is setup

Reply:
You're right. :confused: